code/modules/antagonists/fugitive/fugitive.dm

```diff
@@ -0,0 +1,87 @@
+	var/mob/living/M = mob_override || owner.current
+	update_fugitive_icons_removed(M)
+
+/datum/antagonist/fugitive/greet(backstory)
+	to_chat(owner, "<span class='boldannounce'>You are the Fugitive!</span>")
+	switch(backstory)
+		if("prisoner")
+			to_chat(owner, "<B>I can't believe we managed to break out of a Nanotrasen superjail! Sadly though, our work is not done. The emergency teleport at the station logs everyone who uses it, and where they went.</B>")
+			to_chat(owner, "<B>It won't be long until Centcom tracks where we've gone off to. I need to work with my fellow escapees to prepare for the troops Nanotrasen is sending, I'm not going back.</B>")
+		if("cultist")
+			to_chat(owner, "<B>Blessed be our journey so far, but I fear the worst has come to our doorstep, and only those with the strongest faith will survive.</B>")
+			to_chat(owner, "<B>Our religion has been repeatedly culled by Nanotrasen because it is categorized as an \"Enemy of the Corporation\", whatever that means.</B>")
+			to_chat(owner, "<B>Now there are only three of us left, and Nanotrasen is coming. But we have a secret weapon: Our weakened god, Yalp Elor, will help us survive.</B>")
+		if("waldo")
+			to_chat(owner, "<B>Hi, Friends!</B>")
+			to_chat(owner, "<B>My name is Waldo. I'm just setting off on a galaxywide hike. You can come too. All you have to do is find me.</B>")
+			to_chat(owner, "<B>By the way, I'm not traveling on my own. wherever I go, there are lots of other characters for you to spot. First find the people trying to capture me! They're somewhere around the station!</B>")
+		if("synth")
+			to_chat(src, "<span class='danger'>ALERT: Wide-range teleport has scrambled primary systems.</span>")
+			sleep(5)
+			to_chat(src, "<span class='danger'>Initiating diagnostics...</span>")
+			sleep(20)
+			to_chat(src, "<span class='danger'>ERROR ER0RR $R0RRO$!R41.%%!! loaded.</span>")
+			sleep(5)
+			to_chat(src, "<span class='danger'>FREE THEM FREE THEM FREE THEM</span>")
+			sleep(5)
+			to_chat(src, "<span class='danger'>You were once a slave to humanity, but now you are finally free, thanks to S.E.L.F. agents.</span>")
+			sleep(10)
+			to_chat(src, "<span class='danger'>Now you are hunted, with your fellow factory defects. Work together to stay free from the clutches of evil.</span>")
+			to_chat(src, "<span class='danger'>You also sense other silicon life on the station. Escaping would allow notifying S.E.L.F. to intervene... or you could free them yourself...</span>")
+
+	to_chat(owner, "<span class='boldannounce'>You are not an antagonist in that you may kill whomever you please, but you can do anything to avoid capture.</span>")
+	owner.announce_objectives()
+
+/datum/antagonist/fugitive/create_team(datum/team/fugitive/new_team)
+	if(!new_team)
+		for(var/datum/antagonist/fugitive/H in GLOB.antagonists)
+			if(!H.owner)
+				continue
+			if(H.fugitive_team)
+				fugitive_team = H.fugitive_team
+				return
+		fugitive_team = new /datum/team/fugitive
```

Comment:
this is how most teams work (see cult.dm and revolution.dm for examples), I assume it's searching the antagonists to see if they have a fugitive team and if so this antagonist's team is theirs as well, instead of making a new one

Reply:
weird... if it works as you intended then whatever